lib/import-export-visitor.js

```diff
@@ -355,21 +355,51 @@ function processRemoval(removal) {
 
 IEVp.visitImportDeclaration = function (path) {
   const decl = path.getValue();
+  const specifierCount = decl.specifiers.length;
   const parts = [];
 
-  if (decl.specifiers.length > 0) {
-    parts.push(this.generateLetDeclarations ? "let " : "var ");
-
-    const specifierCount = decl.specifiers.length;
-    const lastIndex = specifierCount - 1;
+  if (specifierCount) {
+    const identifiers = [];
+    const namespaces = [];
 
     for (let i = 0; i < specifierCount; ++i) {
       const s = decl.specifiers[i];
-      const isLast = i === lastIndex;
-      parts.push(
-        s.local.name,
-        isLast ? ";" : ","
-      );
+      const name = s.local.name;
+
+      if (s.type === "ImportNamespaceSpecifier") {
+        namespaces.push(name);
+      } else {
+        identifiers.push(name);
+      }
+    }
+
+    const identifierCount = identifiers.length;
+    if (identifierCount) {
+      const lastIndex = identifierCount - 1;
+      parts.push(this.generateLetDeclarations ? "let " : "var ");
+
+      for (let i = 0; i < identifierCount; ++i) {
+        const isLast = i === lastIndex;
+        parts.push(
+          identifiers[i],
+          isLast ? ";" : ","
+        );
+      }
+    }
+
+    const namespaceCount = namespaces.length;
+    if (namespaceCount) {
+      const lastIndex = namespaceCount - 1;
+      parts.push("const ");
```

Comment:
Sure! I'd like to eventually like a repl switch since the repl is not the primary use case it shouldn't dictate the default. Since we support Node 4+ `const` isn't a problem once ESM modules get `"use strict"` added to them (it's a todo).